apps/projects/app/components/Form/Input/DropDownOptionsInput.js

```diff
@@ -1,17 +1,18 @@
 import PropTypes from 'prop-types'
 import React from 'react'
 import styled from 'styled-components'
-import { IconAdd, IconRemove, TextInput, theme, unselectable } from '@aragon/ui'
+import { IconRemove, TextInput, theme, unselectable, Button } from '@aragon/ui'
 
 class DropDownOptionsInput extends React.Component {
   static propTypes = {
     input: PropTypes.string.isRequired,
     name: PropTypes.string.isRequired,
     onChange: PropTypes.func.isRequired,
     values: PropTypes.array.isRequired,
+    allOptions: PropTypes.array.isRequired,
   }
 
-  addOption = () => {
+  addOption2 = () => {
```

Comment:
This is an older version, left there for reference, but now I think I should have just commented it out or better yet just deleted it, whole addOption2 function. Yes, it can be removed.